modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/AutocompleteFilter.js

```diff
@@ -94,21 +106,34 @@ const getSelectedItemsLabel = ({selectedData}) => {
 	);
 };
 
-const getOdataString = ({id, selectedData, selectionType}) => {
+const getOdataString = ({entityFieldType, id, selectedData, selectionType}) => {
 	const {exclude, itemsValues} = selectedData;
 
 	if (itemsValues?.length) {
 		const values = itemsValues.map((item) => item.value);
 
+		if (entityFieldType === 'string') {
+			return composeStringEntityFieldMultipleValuesOdata(
+				id,
+				values,
+				exclude
+			);
+		}
+
 		return selectionType === 'multiple'
-			? composeMultipleValuesOdataString(id, values, exclude)
+			? composeCollectionEntityFieldMultipleValuesOdata(
+					id,
```

Comment:
I don't fully see why this is called `enbtotyType`when it affects the operator we are using to generate the odata query. WE may use a different naming denoting the real purpose to avoid assumptions that are hard to track from the PoV of the general-purpose component.

Also, I think there's no need to restrict the operator usage to the multiple selection case. How about considering `entityFiledType`as another argument of a more general `composeMultipleValuesOdataString` that would be "operator" aware when it comes to generate the query for the values? 

This way we could do the same (i.e. pass the `entityType`) to the "single value" variant to support single selection from picklist fields.